<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabWidget</class>
 <widget class="QWidget" name="tabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>1041</width>
     <height>750</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="TrialStructureBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>501</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Trial Structure</string>
    </property>
    <widget class="QLabel" name="label_131">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Fixation Time</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_132">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Stimulus Time</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_133">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>120</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Correct</string>
     </property>
    </widget>
    <widget class="QLabel" name="trial_timer">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_134">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>47</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Trial Time</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_135">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>120</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Incorrect</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="stimulus_time">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>80</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>60.000000000000000</double>
     </property>
     <property name="value">
      <double>60.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="fixation_time">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>50</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="accelerated">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="correct_to">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>140</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>3.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="incorrect_to">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>140</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>3.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
     <property name="value">
      <double>1.500000000000000</double>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_56">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_68">
      <item>
       <widget class="QLabel" name="label_136">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>#Valid Trials:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="valid_trials">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_57">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_69">
      <item>
       <widget class="QLabel" name="label_137">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>#Correct Trials:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="correct_trials">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_58">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_70">
      <item>
       <widget class="QLabel" name="label_138">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>#Incorrect Trials</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="incorrect_trials">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_59">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>112</width>
       <height>21</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_71">
      <item>
       <widget class="QLabel" name="label_139">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>#Noresponse Trials</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="noresponse_trials">
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_140">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Stimulus</string>
     </property>
    </widget>
    <widget class="QLabel" name="current_stimulus">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_60">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_72">
      <item>
       <widget class="QLabel" name="label_141">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>#Total Attempts:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="attempt_trials">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_142">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>170</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ITI Interval</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="correct_iti">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>170</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="accelerated">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
     <property name="value">
      <double>0.250000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_143">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>140</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Timeout</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="incorrect_iti">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>170</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="accelerated">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
     <property name="value">
      <double>0.750000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="RewardBox_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>490</y>
      <width>171</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Reward</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_9">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_73">
       <item>
        <widget class="QLabel" name="label_144">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Volume (in uL)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="reward_volume">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="keyboardTracking">
          <bool>true</bool>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>1.500000000000000</double>
         </property>
         <property name="maximum">
          <double>3.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>3.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_17">
       <item>
        <widget class="QPushButton" name="reward_left">
         <property name="text">
          <string>Left Reward</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reward_right">
         <property name="text">
          <string>Right Reward</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_74">
       <item>
        <widget class="QLabel" name="label_145">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Total</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="total_reward">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_18">
       <item>
        <widget class="QPushButton" name="toggle_left_reward">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Left Valve Open</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="toggle_right_reward">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Right Valve Open</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_61">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>590</y>
      <width>101</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="SessionControlBox_2">
     <item>
      <widget class="QLCDNumber" name="session_timer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
border-color: rgb(255, 0, 0);
color: rgb(255, 0, 0);</string>
       </property>
       <property name="intValue" stdset="0">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pause_experiment">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stop_experiment">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>End</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="close_experiment">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>21</y>
      <width>1001</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTabWidget" name="BehaviorMonitor_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="elideMode">
        <enum>Qt::ElideLeft</enum>
       </property>
       <widget class="QWidget" name="tab_25">
        <attribute name="title">
         <string>Video Monitor</string>
        </attribute>
        <widget class="QLabel" name="video_stream">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>441</width>
           <height>410</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_26">
        <attribute name="title">
         <string>Wheel Tracker</string>
        </attribute>
        <widget class="QGraphicsView" name="graphicsView_9">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>451</width>
           <height>410</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="TaskMonitor_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab1_9">
        <attribute name="title">
         <string>Accuracy</string>
        </attribute>
        <widget class="PlotWidget" name="accuracy_plot" native="true">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>470</width>
           <height>410</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab2_9">
        <attribute name="title">
         <string>Psychometric Function</string>
        </attribute>
        <widget class="PlotWidget" name="psychometric_plot" native="true">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>470</width>
           <height>410</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab3_9">
        <attribute name="title">
         <string>Total Trials</string>
        </attribute>
        <widget class="PlotWidget" name="trial_distribution" native="true">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>470</width>
           <height>410</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_27">
        <attribute name="title">
         <string>Reaction Time</string>
        </attribute>
        <widget class="PlotWidget" name="rt_distribution" native="true">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>470</width>
           <height>410</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
